### DISCUSSION TITLE
Problems with Pen Input

### DISCUSSION THREAD

**Discussion by topse** [xournalpp/xournalpp#3373] [General]
Hi,
I am trying to use Xournal++ with a HP Envy Notebook. It has a touchscreen and a pen with pressure sensitivity.

Before Xournal++ I was using a proprietry software to make handwritten notes. Now I see that the pen on the same device is less "sensitive" with Xournal++ than with the other tool.

When I try to write, I experience that the letters are much more edgy than with the other tool. When writing small symbols, they are often -- not always -- degredaded to simple lines so that they cannot be read.

I have tried to play around with the settings for pen, but there was no change at all. Also the pressure sensitivity does not seem to work in Xournal++, but it did work in "the other tool".

Is there anything I can do to find out, what the problem is?

Thanks and best Regards,
       Tobias

**Comment by LittleHuba** [comment]
You can try the GTK demos to see if the input is as bad there as it is in Xournal++. If that is the case, you'll have to open a bug over at the GTK bug tracker.

**Comment by topse** [comment]
Sorry -- I am sure, that is a noob question -- but where do I get those demos? On the GTK-Website I only find things to compile...?

Thanks again :-/

**Comment by Technius** [comment]
Which OS are you using? And what version of Xournal++?

**Reply by topse** [reply]
Hi,
Sorry for the late reply. I am using Windows 10 and the latest stable version of Xournal++ 1.1.0.

Best Regards,
       Tobias

**Reply by Technius** [reply]
Are you using the hotfix version? If not, you might see some improvements by using the hotfix.

**Reply by topse** [reply]
Hotfix solved the problem... I think I may not have seen the newer version below the "old" one...

Thank you very much and Best Regards,
       Tobias

**Reply by LittleHuba** [reply]
Yeah for some reason GitHub sorted the releases this way. I'll add a note in the original release.

**Reply by topse** [reply]
Maybe just make 1.1.1?

**Reply by LittleHuba** [reply]
We are working on that but there are still some PRs open we would like to include.
